### ISSUE TITLE
Custom "Only rendered" text

### ISSUE THREAD

**Issue by frafra** [watergis/maplibre-gl-legend#21]
It would be useful to allow customizing the "Only rendered" string, so that other languages can be supported as well:
https://github.com/watergis/maplibre-gl-legend/blob/de661ee8e05cfa85242c10fb3e0aced35a2e1ca9/packages/maplibre-gl-legend/src/lib/index.ts#L334C38-L334C38

**Comment by JinIgarashi** [comment]
@frafra I agree. Please feel free to create a PR to this repo. I am happy to merge it.
